### ISSUE TITLE
Latest version crashes

### ISSUE THREAD

**Issue by bandrel** [garrettfoster13/pre2k#18]
```                              ___    __
                              /'___`\ /\ \
 _____   _ __    __          /\_\ /\ \\ \ \/'\
/\ '__`\/\`'__\/'__`\ _______\/_/// /__\ \ , <
\ \ \L\ \ \ \//\  __//\______\  // /_\ \\ \ \\`\
 \ \ ,__/\ \_\\ \____\/______/ /\______/ \ \_\ \_\
  \ \ \/  \/_/ \/____/         \/_____/   \/_/\/_/
   \ \_\                                      v3.1
    \/_/
                                            @garrfoster
                                            @Tw1sm

╭──────────────────────────── Traceback (most recent call last) ────────────────────────────╮
│ /usr/local/lib/python3.11/dist-packages/pre2k/lib/commands/auth.py:35 in main             │
│                                                                                           │
│   32 │   │   │   │   │   ldaps=ldaps, binding=binding, kerberos=kerberos, no_pass=no_pass │
│   33 │   │   │   │   │   outputfile=outputfile, stop_on_success=stop_on_success, save=sav │
│   34 │   │   │   │   │   empty_pass=empty_pass, sleep=sleep, jitter=jitter, threads=threa │
│ ❱ 35    pre2k.run()                                                                       │
│                                                                                           │
│ /usr/local/lib/python3.11/dist-packages/pre2k/lib/pre_2k.py:71 in run                     │
│                                                                                           │
│    68 │   │   │   if self.binding:                                                        │
│    69 │   │   │   │   channel_binding = dict(channel_binding=ldap3.TLS_CHANNEL_BINDING)   │
│    70 │   │   │   try:                                                                    │
│ ❱  71 │   │   │   │   ldap_server, ldap_session = init_ldap_session(target_dom=self.targe │
│    72 │   │   │   except ldap3.core.exceptions.LDAPSocketOpenError as e:                  │
│    73 │   │   │   │   if 'invalid server address' in str(e):                              │
│    74 │   │   │   │   │   logger.error (f'Invalid server address - {self.domain}')        │
│                                                                                           │
│ /usr/local/lib/python3.11/dist-packages/pre2k/lib/ldap.py:73 in init_ldap_session         │
│                                                                                           │
│    70 │                                                                                   │
│    71 │   if ldaps:                                                                       │
│    72 │   │   try:                                                                        │
│ ❱  73 │   │   │   return init_ldap_connection(target, ssl.PROTOCOL_TLSv1_2, domain, usern │
│    74 │   │   except ldap3.core.exceptions.LDAPSocketOpenError:                           │
│    75 │   │   │   return init_ldap_connection(target, ssl.PROTOCOL_TLSv1, domain, usernam │
│    76 │   else:                                                                           │
╰───────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: init_ldap_connection() got an unexpected keyword argument 'target_dom'
```

Roll back to the 3.0 version works fine. Tried adding a target domain and still didnt work

**Comment by garrettfoster13** [comment]
Thanks I'll take a look sometime today. Really didn't want to have to lab it lol

**Comment by garrettfoster13** [comment]
Fixed in #19
